pkg/apis/openebs.io/v1alpha1/register.go

```diff
@@ -17,10 +16,11 @@ func Resource(resource string) schema.GroupResource {
 }
 
 var (
+	// SchemeBuilder is the scheme builder with scheme init functions to run for this API package
 	SchemeBuilder runtime.SchemeBuilder
-	// localSchemeBuilder and AddToScheme will stay in k8s.io/kubernetes.
+	// AddToScheme is a global function that registers this API group & version to a scheme
+	AddToScheme        = SchemeBuilder.AddToScheme
 	localSchemeBuilder = &SchemeBuilder
-	AddToScheme        = localSchemeBuilder.AddToScheme
```

Comment:
@cartinez i think you are getting confused :slightly_smiling_face: please do this change.... 
see the order 
```sh
// SchemeBuilder is the scheme builder with scheme init functions to run for this API package	
SchemeBuilder      runtime.SchemeBuilder

// AddToScheme is a global function that registers this API group & version to a scheme	
localSchemeBuilder = &SchemeBuilder
AddToScheme   = localSchemeBuilder.AddToScheme

```

Reply:
oh boy, I'm doing a mess, sorry